src/tools/tests/schema2kotlin-test.ts

```diff
@@ -160,9 +160,9 @@ describe('schema2kotlin', () => {
       `,
       `class Handles : HandleHolderBase(
         "P",
-        mapOf("h1" to P_H1_0)
+        mapOf("h1" to Accommodation)
     ) {
-        val h1: ReadSingletonHandle<Tuple3<Reference<P_H1_0>, Reference<P_H1_1>, Reference<P_H1_2>>> by handles
+        val h1: ReadSingletonHandle<Tuple3<Reference<Person>, Reference<Accommodation>, Reference<Address>>> by handles
```

Comment:
Please add a test for non schema-name based ones, e.g.:
```
particle P
  h1: reads Person {name: Text}
  h2: reads Person {age: Number}
```

As discussed f2f I think we should probably combine the tests for aliases with these tests for Handle class generator, as these things go together. You don't have to do it now and you actually don't have to do it at all, I may write a PR sometime later to do it.

Reply:
Let's do this as a separate PR.